<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetString</class>
 <widget class="QDialog" name="GetString">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../dchat.qrc">
    <normaloff>:/dchatlogo.png</normaloff>:/dchatlogo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="textEnter">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../dchat.qrc"/>
 </resources>
 <connections/>
</ui>
